integration-tests/src/test/java/org/wildfly/prospero/it/installationmanager/CreateSnapshotTest.java

```diff
@@ -66,12 +67,14 @@ public void createSnapshot() throws Exception {
     @Test
     public void generateApplyCommand() throws Exception {
         final Path channelsFile = MetadataTestUtils.prepareChannel(CHANNEL_BASE_CORE_19);
-        final StringBuffer expected = new StringBuffer("prospero");
+        final StringBuffer expected = new StringBuffer("\"")
+                .append(outputPath.resolve("bin").resolve(DistributionInfo.DIST_NAME));
         if (System.getProperty("os.name").toLowerCase(Locale.getDefault()).contains("windows")) {
             expected.append(".bat");
         } else {
             expected.append(".sh");
         }
+        expected.append("\"");
```

Comment:
@yersan '=' needs to be replaced with ' ' in the expected command below

Reply:
ups, right, I forgot to run the test after removing the "=". Replaced.